src/test/java/org/broadinstitute/hellbender/tools/spark/pipelines/BQSRPipelineSparkIntegrationTest.java

```diff
@@ -132,4 +132,19 @@ public void testBlowUpOnBroadcastIncompatibleReference() throws IOException {
                 UserException.Require2BitReferenceForBroadcast.class);
         spec.executeTest("testBQSR-" + params.args, this);
     }
+
+    @Test(expectedExceptions = UserException.MissingReferenceDictFile.class)
+    public void testMissingReferenceDictFileCatch() throws IOException {
```

Comment:
It's best not to bury tests for engine features in tests for tools like this (BQSR, in this case). Recommend moving to a new class `GATKSparkToolIntegrationTest` in the `engine.spark` package.